src/RawPOSIX/src/safeposix/syscalls/fs_calls.rs

```diff
@@ -780,6 +780,44 @@ impl Cage {
         
     }
 
+    pub fn dup3_syscall(&self, old_virtualfd: i32, new_virtualfd: i32, flags: i32) -> i32 {
```

Comment:
Somewhere here, I'd recommend having a link to the manpage for the call.   I'd recommend this as a common pattern for all of the system calls, so this isn't meant to be specific...